CRM/Contribute/Form/Contribution/Main.php

```diff
@@ -995,7 +995,7 @@ public static function formRule($fields, $files, $self) {
       // @todo - processAmount is to be deprectated - can we use getTotalAmount or
       // a function of self->order here?
       CRM_Price_BAO_PriceSet::processAmount($self->_values['fee'],
-        $fields, $lineItem
```

Comment:
this one was not instantiated it - you can get away with that on pass-by-reference